src/videojs.ads.js

```diff
@@ -246,6 +268,37 @@ var
       // just resume playback at the current time.
       player.play();
     }
+
+    i = suppressedTracks.list.length;
+    if (tracks.addTrack_) {
+      while (i--) {
+        track = suppressedTracks.list[i];
+        tracks.addTrack_(track);
```

Comment:
The issue with that is that tracks added with `addTextTrack` in native mode cannot be removed, so, we'd be stuck with just changing the mode of text tracks' mode to be 'disabled', in which case, we don't need to snapshot anything except the mode's of the tracks.